podium-gateway/src/main/webapp/app/shared/request/request-review-panel/request-review-panel.component.ts

```diff
@@ -51,27 +56,22 @@ export class RequestReviewPanelComponent implements OnInit, OnDestroy {
         return foundStyle ? foundStyle.style : 'tag-default';
     }
 
-    private getLastReviewFeedback() {
-        // get the latest start date of review rounds
-        let _lastReviewRoundDate = new Date(Math.max.apply(null, this.reviewRounds.map((reviewRound) => {
-            return new Date(reviewRound.startDate);
-        })));
-
-        // get the latest round
-        let _lastReviewRound = this.reviewRounds.find((reviewRound) => {
-            return new Date(reviewRound.startDate).getTime() === _lastReviewRoundDate.getTime();
-        });
-
-        // return feedback of last review round
-        if (_lastReviewRound && _lastReviewRound.endDate == null) {
-            return _lastReviewRound.reviewFeedback;
-        }
-        return null;
+    isReviewer() {
+        return this.requestAccessService.isReviewerFor(this.request);
     }
 
     ngOnInit(): void {
-        if (this.reviewRounds.length) {
-            this.lastReviewFeedback = this.getLastReviewFeedback();
+        if (this.request.reviewRounds.length) {
+            if (this.requestAccessService.isReviewerFor(this.request)) {
+                this.principal.identity().then((account) => {
+                    this.currentUser = account;
```

Comment:
used in this.requestService.getLastReviewFeedbackByUser(this.request, this.currentUser)